templates/master/01-master-kubelet/_base/files/criometricsproxy-config.yaml

```diff
@@ -0,0 +1,11 @@
+mode: 0644
+path: "/etc/kubernetes/crio-metrics-proxy.cfg"
+contents:
+  inline: |-
+    authorization:
+      static:
+        - resourceRequest: false
+          path: /metrics
+          verb: get
+          user:
+            name: system:serviceaccount:openshift-monitoring:prometheus-k8s
```

Comment:
is this using the openshift-monitoring SA?

In our other kube-rbac-proxy pods we specify which `serviceAccountName:` we want to use. Where is that happening in this PR?

Reply:
This new kube-rbac-proxy-crio service does not have access to service account secrets due to being a static pod. Instead, we are using client side certificate validation and only allowing the `system:serviceaccount:openshift-monitoring:prometheus-k8s` user.